specs/mediawiki/v1/content.yaml

```diff
@@ -75,7 +70,7 @@ paths:
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}/latest
 
-  /{module:page}/{title}/html/:
+  /{module:page}/html/{title}/:
```

Comment:
Re: using the Accept header to choose the resource representation(s), see: https://developer.github.com/v3/media/#comment-body-properties